src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -83,17 +83,22 @@ export const fetchDictionaries = () => (dispatch) => {
 };
 
 export const searchDictionaries = searchItem => (dispatch) => {
-  dispatch(isFetching(true));
-  return api.dictionaries
-    .searchDictionaries(searchItem)
-    .then((payload) => {
-      dispatch(isSuccess(payload));
-      dispatch(isFetching(false));
-    })
-    .catch(error => () => {
-      dispatch(isErrored(error.payload.data));
-      dispatch(isFetching(false));
-    });
+  if (navigator.onLine){
```

Comment:
`navigator.onLine` will be `true` so long as the machine is connected to a `network` even in the absence of `internet`. Won't it give us false positives?